frontend/eslint-plugin/rules/no-testonly.js

```diff
@@ -14,6 +14,10 @@ function isCallOfOnly(node) {
   return node.callee.property && node.callee.property.name === 'only'
 }
 
+function isItOnlyCall(node) {
+  return isCallExpression(node) && isMethodOf('it', node) && isCallOfOnly(node)
+}
+
```

Comment:
This could be added to the existing `isTestOnlyCall` and they could be combined.

`isMethodOf(['test', 'it', 'describe', 'fixture'], node)`